anthos-bm-apigee/README.md

```diff
@@ -0,0 +1,81 @@
+
+### Pre-requisites
+
+- A workstation with access to internet _(i.e. Google Cloud APIs)_ with the following installed
+  - [Git](https://www.atlassian.com/git/tutorials/install-git)
+  - [Google Cloud SDK](https://cloud.google.com/sdk/docs/install)
+  - [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) (>= v0.15.5, < v1.2)
+
+- A [Google Cloud Project](https://console.cloud.google.com/cloud-resource-manager?_ga=2.187862184.1029435410.1614837439-1338907320.1614299892) _(in which the resources for the setup will be provisioned)_
+
+- Enable Compute Engine API Services and Apigee API Services.
+
+- Ensure that the default Compute Engine developer account has Editor role. You can also create a new service account for the compute engine with Editor and specify the service account name in terraform variables.
+
+![Default Service Account](docs/images/default_service_account.png)
+
+- A [Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) in the project that satisfies **one** of the following requirements and its **[key file downloaded](docs/create_sa_key.md)** to the workstation:
+    - The Service Account has `Owner` and `Apigee Organization Administrator` permissions
+    - The Service Account has both `Editor`, `Project IAM Admin` and `Apigee Organization Administrator` permissions
+The scripts to create service account and key creation is also mentioned in Quickstart module.
+- Organizational Policy Constraints
+      Follwowing list of Policy needs to be enabled for the organizations. If these are not enabled at Organizations, please consider them for the project.
+
+      |  Policy Name                                 | Constraint Name                                   | Effective Polciy |
+      |  ------------------------------------------- | ------------------------------------------------- | ---------------- |
+      | Disable service account creation             | constraints/iam.disableServiceAccountCreation	   | Not Enforced     |
+      | Disable service account key creation         | constraints/iam.disableServiceAccountKeyCreation	 | Not enforced     |
+      | Restrict VM IP Forwarding                    | constraints/compute.vmCanIpForward	         | Allowed All      |
+      | Define allowed external IPs for VM instances | constraints/compute.vmExternalIpAccess	           | Allowed All      |
+      | Shielded VMs                                 | constraints/compute.requireShieldedVm	           | Not Enforced     |
+      | Require OS Login                             | constraints/compute.requireOsLogin	               | Not Enforced     |
+      | Skip default network creation                | constraints/compute.skipDefaultNetworkCreation	   | Not Enforced     |
```

Comment:
Please check the rendered view to ensure if the table is shown properly. Now it looks like this:

![image](https://user-images.githubusercontent.com/7249208/194353772-52b8bd43-2d36-41bd-8c11-2a5dffed58f3.png)

Reply:
Alignment is fixed.